cypress/e2e/02_test_items.cy.js

```diff
@@ -0,0 +1,198 @@
+/*
+this should be 01, not 02 (because testing entities requires items to be working
+ */
+
+describe("ffg-star-wars create items", () => {
+  beforeEach(() => {
+    cy.join();
+    cy.waitUntilReady();
+    cy.get('#sidebar-tabs > [data-tab="items"]').click();
+  });
+
+  it("sets up the folder", () => {
+    // delete the folder
+    cy.get("#items > .directory-list header h3").contains("test items").rightclick();
+    cy.get("#context-menu > .context-items > .context-item").contains("Delete All").click();
+    cy.get(".window-content > .dialog-buttons > .yes").click().should("not.exist");
+
+    // create the folder
+    cy.get('#items > .directory-header > .header-actions > .create-folder').click().then(function () {
+      cy.get(':nth-child(3) > .form-fields > input').type('test items');
+      cy.get('form > button').click().then(function () {
+        // wait for the window to go away
+        cy.get('[type="color"]').should('not.be.visible');
+      });
+    });
+  });
+  /*
+  it("creates an ability", () => {
```

Comment:
Are these meant to all be commented out?

Reply:
they're not used so I didn't uncomment them, but they do work. I can uncomment if desired

Reply:
:shrug: Happy to call it a work in progress.